test/CryptoMarriage.test.js

```diff
@@ -0,0 +1,194 @@
+const CM = artifacts.require("CryptoMarriage");
+const truffleAssert = require("truffle-assertions");
```

Comment:
truffle tests! So old school. If you feel like experiment with more modern stuff, you'd probably have a better time with:

- hardhat (kinda what everyone is using right now)
- dapptools (write tests in Solidity! super fast)
- foundry, which is brand new and sounds pretty exciting (https://mirror.xyz/sha.eth/6Mn3HjrqKLhHzu2balLPv4SqE5a-oEESl4ycpRkWFsc)
-

Reply:
haha I copied what I saw elsewhere... foundry looks amazing ! Will definitely try that next time